src/applications/lgy/coe/containers/introduction-content/COEStatuses/COEAutomatic.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const COEAutomatic = () => (
+  <>
+    <va-alert status="success">
+      <h3 slot="headline">Congratulations on your automatic COE</h3>
```

Comment:
Should this be an h2 with `vads-u-font-size--h3`? I think the heading right before this is an h1.

Reply:
Interestingly it doesn't, looks like the styling comes from the `va-alert` component itself

<img width="298" alt="Screen Shot 2021-09-17 at 12 51 17 PM" src="https://user-images.githubusercontent.com/1899695/133845778-8fadc50a-6b6e-451f-86dd-a5e478ca50ed.png">